.github/workflows/check-pr-branch.yml

```diff
@@ -4,14 +4,13 @@ on:
   pull_request:
     types: ['*']
     branches:
-      - 'master'
+      - '**'
```

Comment:
Is this better than limiting this to only run on PRs into master?

Reply:
Yeah, that is the problem - it wasn't re-running when the base branch changed away from master, leading to it being permanently red. See: https://github.com/iron-fish/ironfish/pull/4027